cognite_toolkit/_cdf_tk/commands/_purge.py

```diff
@@ -85,6 +91,80 @@ def as_id(self) -> Hashable:
         return self.as_id_fun(self.item)
 
 
+@dataclass
+class ToDelete(ABC):
```

Comment:
I moved out this class such that it can handled the differences in how the resources are translated into IDs.